week1/assignment/todo-tan/list.rb

```diff
@@ -0,0 +1,79 @@
+require_relative "item"
+
+class List
+  attr_accessor :name
+  attr_accessor :items
+
+  def initialize(name, items = [])
+    @name = name
+    @items = items
+  end
+
+  def add(item)
+    @items << item
+  end
+
+  def remove_at(index)
+    @items.delete_at(index)
+  end
+
+  def remove_by_name(name)
+
+  end
+
+  def mark_done_at(index)
+    @items[index].mark_done!
+  end
+
+  def find_index_by_name(name)
+    items.index {|item| item.name == name}
+  end
+
+  def done
+    @items.select { |item| item.done? }
+  end
+
+  def undone
+    @items.select { |item| !item.done? }
+  end
+
+
+  def display_done
+    puts "Displaying DONE items:"
+    @items.each_with_index do |item, index|
```

Comment:
you can even do `done.each_with_index` here because you already have a `done` method above. maybe the method name could be `done_items` to be clearer